### PR TITLE
[5518] test quoted arguments in imeta interactive mode (master)

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: scripts/irods/test/test_imeta_interactive.py [hunk 1]
```diff
@@ -186,6 +186,23 @@ def test_ls_no_args(self):
...
+    def test_imeta_quoted_arguments__5518(self):
+        self.get_output()
+        data_name = self.test_data_path_base + '1'
+        self.imeta_p.stdin.write('adda -d {} "with spaces" "c\td" "e \tf"\n'.format(data_name))
+        self.imeta_p.stdin.flush()
+        (out, err) = self.get_output()
+        self.assertEqual(out, 'imeta>')
+        self.admin.assert_icommand(['iquest','--no-page', '%s/%s/%s',
+                                    "select META_DATA_ATTR_NAME, META_DATA_ATTR_VALUE, META_DATA_ATTR_UNITS where DATA_NAME = '{}' "
+                                    " and META_DATA_ATTR_NAME like 'with%spaces'".format( data_name.split('/')[-1] )
```
Comment (trel) [inline scripts/irods/test/test_imeta_interactive.py:198]: don't we want this to be for precise?

`META_DATA_ATTR_NAME = 'with spaces'`

**Comment by trel** [comment]
and one codacy thing

File: scripts/irods/test/test_imeta_interactive.py (hunk 1 above)
Comment (d-w-moore) [reply scripts/irods/test/test_imeta_interactive.py:198]: You mean make the regex match more specific? Could do that.

Comment (trel) [reply scripts/irods/test/test_imeta_interactive.py:198]: no, the genquery itself.   i don't think it really matters, but it threw me off that you're using`like`.

Comment (d-w-moore) [reply scripts/irods/test/test_imeta_interactive.py:198]: While I'm refining the genquery key, I'll make the regex match tighter too.

Comment (d-w-moore) [reply scripts/irods/test/test_imeta_interactive.py]: done

**Pull request merged**